### PR TITLE
Add links to promote database engine calculator

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Fixes netdata/marketing#222

This PR adds a few links to the [database engine calculator](https://learn.netdata.cloud/docs/agent/database/calculator) I built a few weeks ago. In netdata/netdata-learn-docusaurus#133, I changed up the navigation to surface all of this content better.

In addition, I added some manual descriptions to these pages to help boost their SEO value.

##### Component Name

database
docs

### CODE REVIEW

**Comment by mfundul** [comment]
From https://learn.netdata.cloud/docs/agent/database/calculator something looks wrong to me; the part that instructs the user to edit netdata.conf:
```
[global]
    dbengine disk space = 5933
```
This shouldn't be the same as the aggregate disk space needed for all slaves. For `N` slaves this would need to be `5933 / (N+1)`.

File: database/engine/README.md [hunk 1]
```diff
@@ -63,6 +62,14 @@ All DB engine instances, for localhost and all other streaming recipient nodes i
 The above values are the default and minimum values for Page Cache size and DB engine disk space quota. Both numbers are
 in **MiB**. All DB engine instances will allocate the configured resources separately.
 
+[**See our database engine calculator**](https://learn.netdata.cloud/docs/agent/database/calculator) to help you
+correctly set `dbengine disk space` based on your needs. The calculator gives an accurate estimate based on how many
+slave nodes you have, how many metrics your Agent collects, and more.
+
```
Comment (zack-shoylev) [inline database/engine/README.md:68]: Is something missing here?

Comment (joelhans) [reply database/engine/README.md:68]: I forgot to delete this paragraph. Thanks for catching it.

**Comment by zack-shoylev** [comment]
There has also been an ask to have the calculator work the other way: calculate maximum number of metrics based on machine hardware.

**Comment by joelhans** [comment]
@zack-shoylev Who is this ask coming from? Internal or a user? I could probably write a second calculator to do this, but it would need to be a separate task and it would probably take me a week or two. Is it a blocker to adding these links and promoting the existing calculator more?

**Comment by zack-shoylev** [comment]
Not a blocker, from user. We can discuss more later this week. It will have to be a separate issue, yes

**Pull request merged**